kubernetes-services/work-scheduler/env-defaults

```diff
@@ -0,0 +1,61 @@
+# Port the health check server listens on
+PORT=5000
+
+# Fake credentials for localstack
+AWS_ACCESS_KEY_ID=LOCALSTACK
+AWS_SECRET_ACCESS_KEY=LOCALSTACK
+
+AWS_DEFAULT_REGION=us-west-2
+
+NODE_ENV=production
```

Comment:
Normally we default things for running locally - why do we set it to production? Note to self that if this is just an indicator to use Postgres we should probably stop using NODE_ENV for that.

Reply:
Yes, this is so the work-scheduler will default to using postgres. We actually default things to work with Harmony in a Box (HIAB), so this needs to use postgres by default since HIAB is not set up to share a file system between pods (and therefore can't use sqlite). Your point about not using NODE_ENV for this is a good one, though. Not sure if we want to change that now or later.